notebooks/113-image-classification-quantization/113-image-classification-quantization.ipynb

```diff
@@ -0,0 +1,446 @@
+        "metric_results = pipeline.evaluate(model)\n",
+        "if metric_results:\n",
+        "    for name, value in metric_results.items():\n",
+        "        print(f\"Accuracy of the original model: {name}: {value}\")\n",
+        "\n",
+        "metric_results = pipeline.evaluate(compressed_model)\n",
+        "if metric_results:\n",
+        "    for name, value in metric_results.items():\n",
+        "        print(f\"Accuracy of the optimized model: {name}: {value}\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vQACMfAUo52V"
+      },
+      "source": [
+        "## Compare Performance of the Original and Quantized Models\n",
+        "\n",
+        "Finally, we will measure the inference performance of the FP32 and INT8 models. To do this, we use [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html) - OpenVINO's inference performance measurement tool.\n",
+        "\n",
+        "NOTE: For more accurate performance, we recommended running benchmark_app in a terminal/command prompt after closing other applications. Run benchmark_app -m model.xml -d CPU to benchmark async inference on CPU for one minute. Change CPU to GPU to benchmark on GPU. Run benchmark_app --help to see an overview of all command line options.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "pC0gnO0c9-tI"
+      },
+      "outputs": [],
+      "source": [
+        "# Inference FP16 model (IR)\n",
+        "!benchmark_app -m $ir_model_xml -d CPU -api async"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4VR3-joFu9hH"
+      },
+      "outputs": [],
+      "source": [
+        "# Inference INT8 model (IR)\n",
+        "!benchmark_app -m $compressed_model_xml -d CPU -api async"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Compare results on one picture."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "labels_names = [\"airplane\", \"automobile\", \"bird\", \"cat\", \"deer\", \"dog\", \"frog\", \"horse\", \"ship\", \"truck\"]\n",
+        "all_batches = []\n",
+        "all_labels = []\n",
+        "\n",
+        "for i, batch in enumerate(data_loader):\n",
+        "    all_batches.append(batch[1])\n",
+        "    all_labels.append(batch[0][1])"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "picture_index = 60\n",
+        "\n",
+        "label = all_labels[picture_index]\n",
+        "image = all_batches[picture_index].squeeze().numpy()\n",
+        "image = np.rollaxis(image, 0, 3)\n",
+        "image.shape\n",
```

Comment:
The picture is displayed just because the inference of it is done later. It was suggested before. To make an inference of 4 pictures will take more code which is not as flexible as inference of one picture chosen by the user. So I suggest doing inference only on one picture. Displaying pictures is not the core idea, in my opinion.